lms/static/js/spec/edxnotes/views/tabs/recent_activity_spec.js

```diff
@@ -50,28 +70,135 @@ define([
                 'templates/edxnotes/note-item', 'templates/edxnotes/tab-item'
             ]);
 
-            this.collection = new NotesCollection(notes);
+            this.collection = new NotesCollection(notes, {perPage: 10, parse: true});
             this.tabsCollection = new TabsCollection();
         });
 
         it('displays a tab and content with proper data and order', function () {
             var view = getView(this.collection, this.tabsCollection);
+            Helpers.verifyPaginationInfo(view, "Showing 1-3 out of 3 total", 1, 1);
+            Helpers.verifyPageData(view, this.tabsCollection, tabInfo, recentActivityTabId, notes);
+        });
 
-            expect(this.tabsCollection).toHaveLength(1);
-            expect(this.tabsCollection.at(0).toJSON()).toEqual({
-                name: 'Recent Activity',
-                identifier: 'view-recent-activity',
-                icon: 'fa fa-clock-o',
-                is_active: true,
-                is_closable: false,
-                view: 'Recent Activity'
-            });
-            expect(view.$('#recent-panel')).toExist();
-            expect(view.$('.note')).toHaveLength(3);
-            _.each(view.$('.note'), function(element, index) {
-                expect($('.note-comments', element)).toContainText(notes[index].text);
-                expect($('.note-excerpt', element)).toContainText(notes[index].quote);
-            });
+        it("will not render header and footer if there are no notes", function () {
+            var notes = Helpers.createNotesData(
+                {
+                    numNotesToCreate: 0,
+                    count: 0,
+                    num_pages: 1,
+                    current_page: 1,
+                    start: 0
+                }
+            );
+            var collection = new NotesCollection(notes, {perPage: 10, parse: true});
+            var view = getView(collection, this.tabsCollection);
+            expect(view.$('.search-tools.listing-tools')).toHaveLength(0);
+            expect(view.$('.pagination.pagination-full.bottom')).toHaveLength(0);
+        });
+
+        it("can go to a page number", function () {
+            var requests = AjaxHelpers.requests(this);
+            var notes = Helpers.createNotesData(
+                {
+                    numNotesToCreate: 10,
+                    count: 12,
+                    num_pages: 2,
+                    current_page: 1,
+                    start: 0
+                }
+            );
+            var collection = new NotesCollection(notes, {perPage: 10, parse: true});
+            var view = getView(collection, this.tabsCollection);
+
+            Helpers.verifyPaginationInfo(view, "Showing 1-10 out of 12 total", 1, 2);
```

Comment:
`numNotesToCreate` is the number of notes in the current page in response and `count` will be the total number of notes available. We can't make them same.